Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -237,22 +237,43 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::DoPartialTextureUpdates(Tex
 		{
 			if (entry->hash == entry->CalculateHash())
 			{
+				MathUtil::Rectangle<int> srcrect, dstrect;
+
 				u32 block_offset = (entry->addr - entry_to_update->addr) / block_size;
 				u32 block_x = block_offset % numBlocksX;
 				u32 block_y = block_offset / numBlocksX;
 
-				u32 x = block_x * block_width;
-				u32 y = block_y * block_height;
-				MathUtil::Rectangle<int> srcrect, dstrect;
-				srcrect.left = 0;
-				srcrect.top = 0;
-				dstrect.left = 0;
-				dstrect.top = 0;
+				u32 dst_x = block_x * block_width;
+				u32 dst_y = block_y * block_height;
+				u32 src_x = 0;
+				u32 src_y = 0;
+
+				// If the EFB copy doesn't fully fit, cancel the copy process
```

Comment:
This if will be removed later, and then the use of minimum width and height will make sense. I intentionally made this pr, so it does not support this case, in order to get a stable and reliable basis for the next pr. The other pr3266 did that, but opengl and d3d gave different results, which made no sense. This pr now seems to work exactly as expected and then i'll update pr3266.